dahu/core/app/scripts/templates/views/workspace/actions/move.html

```diff
@@ -1,7 +1,6 @@
-<div class="move">
```

Comment:
keep a container, even if it is a simple div without class.
just remove the class then.